### PR TITLE
Implement new Concept Exercise: tuples

### PR DESCRIPTION
**Pull request by Grenkin1988**
Fixws #863

### CODE REVIEW

File: exercises/concept/tisbury-treasure-hunt/.docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+("one", "two", "three")
+// Tuple of generic types.
+(a, b)
+// Tuple that has mixed types.
+("one", 1, 2.0)
+// Tuple of integer expressions.
+(a + 1, b + 1)
+```
+
+## Obtaining Individual Values
```
[the remark this answers is not in the document]
Comment (Grenkin1988) [reply exercises/concept/tisbury-treasure-hunt/.docs/introduction.md:32]: Sorry, not sure I understand what is required

File: concepts/tuples/about.md [hunk 2]
```diff
@@ -6,15 +6,8 @@ A [tuple][tuple] is a _immutable_ grouping of unnamed but ordered values, possib
...
-
-### Using the `System.Tuple()` constructor:
-
-```fsharp
-let value = System.Tuple<int,int>(1, 2)
-```
-
-### Examples of tuples include pairs, triples, and so on, of the same or different types. Some examples are illustrated in the following code.:
+Tuples can be created using `(<element_1>, <element_2>)` declaration.
+Tuples ca be pairs, triples, and so on, of the same or different types. Some examples are illustrated in the following code.:
```
Comment (ErikSchierboom) [inline concepts/tuples/about.md:10]:
```suggestion
Tuples can be pairs, triples, and so on, of the same or different types. Some examples are illustrated in the following code:
```

File: exercises/concept/tisbury-treasure-hunt/.docs/introduction.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply exercises/concept/tisbury-treasure-hunt/.docs/introduction.md:32]: No problem. I can do a follow-up PR later.

**Comment by ErikSchierboom** [comment]
@Grenkin1988 Happy with the current state of the PR? If so, I can merge this and tweak the introduction phrasing slightly in a follow-up PR (easier that modifying in this PR).

**Comment by Grenkin1988** [comment]
I guess so

**Review by ErikSchierboom — APPROVED** [review]
Let's go! Awesome work @Grenkin1988! 🎉

**Comment by Grenkin1988** [comment]
This was really interesting experience 😸 Hope it wasn't way to terrible 😅

**Pull request merged**